packages/ui-library/src/components/textarea/index.css.ts

```diff
@@ -34,11 +34,13 @@ export const staticStyles = typeSafeNestedCss/*css*/ `
   .blr-textarea-info-container {
     display: flex;
     justify-content: right;
-
-    &.hint, &.error {
+    &.error{
+      justify-content: right;
+    }
+    &.hint{
       justify-content: space-between;
     }
-  }
+  } 
```

Comment:
Seems like your branch is out of sync with develop branch, please fix that and will approve it. Also please remove this extra space at line 43.